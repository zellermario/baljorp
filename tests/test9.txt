create SJ m1 0
create L m2 0
addNeighbor m1 1 m2
addNeighbor m2 2 m1

create Eszkimo e1 m1
create Eszkimo e2 m2
create Kotel k player e1

useobject k e1
pass e1