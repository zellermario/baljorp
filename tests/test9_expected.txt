Fields:
SJ m1 (0) - object: x
	neighbours: [(1,m2)]
L m2 (0) - object: x
	neighbours: [(2,m1)]

Buildings:

Actors:
> Eszkimo e1 (heat: 5) on m1
  Inventory: [k]
> Eszkimo e2 (heat: 5) on m1
  Inventory: []